skfda/exploratory/visualization/representation.py

```diff
@@ -119,63 +121,96 @@ def _get_color_info(
     argument to display the functions wtih a gradient of colors.
-
     Args:
         fdata: functional data set that we want to plot.
-        gradient_values: list of real values used to determine the color
-            in which each of the instances will be plotted.
+        gradient_color_list: list of real values used to determine the color
+            in which each of the instances will be plotted. The size
         max_grad: maximum value that the gradient_list can take, it will be
-            used to normalize the ``gradient_values``. If not
+            used to normalize the gradient_color_list in order to get values
+            thatcan be used in the funcion colormap.__call__(). If not
             declared it will be initialized to the maximum value of
             gradient_list
         min_grad: minimum value that the gradient_list can take, it will be
-            used to normalize the ``gradient_values``. If not
+            used to normalize the gradient_color_list in order to get values
+            thatcan be used in the funcion colormap.__call__(). If not
             declared it will be initialized to the minimum value of
             gradient_list.
-
+        chart (figure object, axe or list of axes, optional): figure over
+            with the graphs are plotted or axis over where the graphs are
+            plotted. If None and ax is also None, the figure is
+            initialized.
+        fig (figure object, optional): figure over with the graphs are
+            plotted in case ax is not specified. If None and ax is also
+            None, the figure is initialized.
+        axes (axis object, optional): axis over where the graphs
+            are plotted. If None, see param fig.
+        n_rows (int, optional): designates the number of rows of the figure
+            to plot the different dimensions of the image. Only specified
+            if fig and ax are None.
+        n_cols(int, optional): designates the number of columns of the
+            figure to plot the different dimensions of the image. Only
+            specified if fig and ax are None.
+    Attributes:
+        gradient_list: normalization of the values from gradient color_list
+            that will be used to determine the intensity of the color
+            each function will have.
     """
 
     def __init__(
         self,
         fdata: FData,
-        gradient_values: Optional[Sequence[float]] = None,
+        gradient_color_list: Union[Sequence[float], None] = None,
         max_grad: Optional[float] = None,
         min_grad: Optional[float] = None,
+        chart: Union[Figure, Axes, None] = None,
+        *,
+        fig: Optional[Figure] = None,
+        axes: Optional[Axes] = None,
+        n_rows: Optional[int] = None,
+        n_cols: Optional[int] = None,
+        n_points: Union[int, Tuple[int, int], None] = None,
+        domain_range: Optional[DomainRangeLike] = None,
+        group: Optional[Sequence[K]] = None,
```

Comment:
Some of these options, such as `group`, `group_colors`, etc could be applied to other plots, such as the DDPlot. Have you thought about it?

Reply:
yes, you mentioned it to me the other day and i answered you